packages/ec-core/src/distributions/wrappers/owned.rs

```diff
@@ -81,6 +82,16 @@ where
     T: Borrow<[U]>,
     U: 'a + Clone,
 {
+    #[rustversion::attr(before(1.81), allow(clippy::unwrap_used))]
+    #[rustversion::attr(
+        since(1.81),
+        expect(
+            clippy::unwrap_used,
+            reason = "At construction time, it was ensured that the slice was non-empty, and that \
+                      the `Uniform` range produces values in range for the slice - this should \
+                      never occur"
```

Comment:
This is currently needed since we can't apply custom attribute macros to expressions ( is nightly only ) and should be moved back once we revert the second commit in 2 months or so.

Reply:
Gotcha – that makes sense. It's a shame that we have to do so much dancing around here, but your use of commits should make it a lot easier.